exec/java-exec/src/main/java/org/apache/drill/exec/ops/QueryContext.java

```diff
@@ -87,6 +89,7 @@ public enum SqlStatementType {OTHER, ANALYZE, CTAS, EXPLAIN, DESCRIBE_TABLE, DES
   /** Stores constants and their holders by type */
   private final Map<String, Map<MinorType, ValueHolder>> constantValueHolderCache;
   private SqlStatementType stmtType;
+  private Collection<StoragePlugin> scannedPlugins;
```

Comment:
I wasn't sure whether this new member is appropriate here.

Reply:
Could you please make it a list of strings and pass it within PlanProperties?